river/evaluate/tracks.py

```diff
@@ -44,36 +44,49 @@ def run(self, model, dataset, n_checkpoints=10):
 
 
 class BinaryClassificationTrack(Track):
-    def __init__(self):
+    def __init__(self, n_samples=10_000):
+        dsets = [
+            getattr(importlib.import_module("river.datasets"), i)().take(k=n_samples)
+            for i in importlib.import_module("river.datasets").__all__
+            if callable(getattr(importlib.import_module("river.datasets"), i))
+            and getattr(importlib.import_module("river.datasets"), i)().task
+            == datasets.base.BINARY_CLF
+        ]
```

Comment:
I'm not 100% sure it's a good idea to import all datasets. Some datasets are really large and will slow down the benchmarks by a ton

Reply:
I'm trying to finish evaluating it right now... and you're right. 
I'm still looking for a way to incorporate the benchmarks in such a way that we can map relevant data streams such as Elec and Covertype on the one hand, but also synthetic data streams with concept drifts. 

But maybe that's anoter issue.

Reply:
You could take a sample of each dataset. Each dataset has a `take(k)` method to select the first `k` rows in the stream.

Overall I think you/we should aim for 1 hour max to evaluate all benchmarks